src/main/java/com/example/demo/database/Orders.java

```diff
@@ -0,0 +1,56 @@
+package com.example.demo.database;
+import jakarta.persistence.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+@Entity
+@Table
+public class Orders {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(nullable = false)
+    private long order_id;
+
+    private long customer_id;
+
+    private int total_amount;
+
+    private String date_time;
```

Comment:
If you have date and time you can use `LocaDate` or `LocalDateTime` for the representation of a time